service/provider/src/main/java/gov/va/vro/service/provider/mas/service/MasCollectionService.java

```diff
@@ -84,25 +86,52 @@ public HealthDataAssessment collectAnnotations(MasAutomatedClaimPayload claimPay
     healthDataAssessment.setEvidence(abdEvidence);
     healthDataAssessment.setVeteranIcn(claimPayload.getVeteranIdentifiers().getIcn());
     healthDataAssessment.setDisabilityActionType(
-        claimPayload.getClaimDetail().getConditions().getDisabilityActionType());
+            claimPayload.getClaimDetail().getConditions().getDisabilityActionType());
     return healthDataAssessment;
   }
 
   public static HealthDataAssessment combineEvidence(
-      HealthDataAssessment lighthouseAssessment, HealthDataAssessment masApiAssessment) {
+          HealthDataAssessment lighthouseAssessment, HealthDataAssessment masApiAssessment) {
     AbdEvidence lighthouseEvidence = lighthouseAssessment.getEvidence();
     AbdEvidence masApiEvidence = masApiAssessment.getEvidence();
     // for now, we just add up the lists
+    log.info("combineEvidence >> LH  : " + lighthouseEvidence != null ? "not null" : "null");
+    log.info("combineEvidence >> MAS : " + masApiEvidence != null ? "not null" : "null");
     AbdEvidence compositeEvidence = new AbdEvidence();
     compositeEvidence.setBloodPressures(
-        merge(lighthouseEvidence.getBloodPressures(), masApiEvidence.getBloodPressures()));
+            merge(
+                    lighthouseEvidence != null ? lighthouseEvidence.getBloodPressures() : null,
+                    masApiEvidence != null ? masApiEvidence.getBloodPressures() : null)
+                    .stream()
```

Comment:
This code is the same for every list so it should be extracted to a common method